apps/src/code-studio/pd/application/teacher/TeacherApplication.jsx

```diff
@@ -33,19 +32,8 @@ const autoComputedFields = [
   'regionalPartnerWorkshopIds',
 ];
 
-const sendFirehoseEvent = (userId, event) => {
-  firehoseClient.putRecord(
-    {
-      user_id: userId,
-      study: 'application-funnel',
-      event: event,
-    },
-    {includeUserId: false}
-  );
-};
-
 const TeacherApplication = props => {
-  const {savedFormData, accountEmail, userId, savedStatus, schoolId} = props;
```

Comment:
If we no longer need the `userId` prop here, do we need it in the `new` method of the `teacher_application_controller.rb`?

Reply:
Good catch- thank you! Removed!